### PR TITLE
kconfig: openthread: Temporarily disable HUK usage

### PR DESCRIPTION
**Pull request by maciejbaczmanski**
Temporarily disable HUK usage until HUK provisioning is implemented

### CODE REVIEW

**Review by canisLupus1313 — APPROVED** [review]

File: subsys/net/openthread/Kconfig [hunk 1]
```diff
@@ -64,6 +64,15 @@ config OPENTHREAD_NRF_SECURITY_PSA
 	select SETTINGS
 	depends on !BUILD_WITH_TFM
 
+if OPENTHREAD_NRF_SECURITY_PSA
+# Temporarily disable HUK usage until HUK provisioning is implemented
+
+choice SECURE_STORAGE_BACKEND_AEAD_KEY
+	default SECURE_STORAGE_BACKEND_AEAD_KEY_HASH_UID
```
Comment (joerchan) [inline subsys/net/openthread/Kconfig:71]:
```suggestion
	default SECURE_STORAGE_BACKEND_AEAD_KEY_HASH_UID if OPENTHREAD_NRF_SECURITY_PSA
```

File: subsys/net/openthread/Kconfig [hunk 2]
```diff
@@ -64,6 +64,15 @@ config OPENTHREAD_NRF_SECURITY_PSA
 	select SETTINGS
 	depends on !BUILD_WITH_TFM
 
+if OPENTHREAD_NRF_SECURITY_PSA
+# Temporarily disable HUK usage until HUK provisioning is implemented
```
Comment (edmont) [inline subsys/net/openthread/Kconfig:68]:
```suggestion
# Temporarily use hash of UID as AEAD Key implementation until Hardware Unique Key provisioning (more secure) is implemented
```

**Review by edmont — APPROVED** [review]
Improved comment suggestion.

File: subsys/net/openthread/Kconfig (hunk 1 above)
Comment (joerchan) [reply subsys/net/openthread/Kconfig:71]: Can make the whole thing shorter when it is just one item inside the `if`.

**Review by edmont — APPROVED** [review]

**Review by Vge0rge — APPROVED** [review]
LGTM

**Pull request merged**